scripts/generators/tsconfig.js

```diff
@@ -92,13 +92,21 @@ function getTsPkgs(subRoot) {
           if (typeof exportPath === "object") {
             exportPath = exportPath.default;
           }
-          if (exportPath.startsWith("./lib") && exportPath.endsWith(".js")) {
+          if (
+            exportPath.startsWith("./lib") &&
+            (exportPath.endsWith(".js") || exportPath.endsWith(".cjs"))
+          ) {
             // remove the leading `.` and trailing `.js`
-            const subExport = _export.slice(1).replace(/\.js$/, "");
+            const subExport = _export
+              .slice(1)
+              .replace(/\.js$/, "")
+              .replace(/\.cjs$/, "");
             const subExportPath = exportPath
               .replace("./lib", "/src")
               .replace(/\.js$/, ".ts")
-              .replace(/\/index\.ts$/, "");
+              .replace(/\.cjs$/, ".cts")
+              .replace(/\/index\.ts$/, "")
+              .replace(/\/index\.cts$/, "");
```

Comment:
```suggestion
              .replace(/\.c?js$/, ".cts")
              .replace(/\/index\.c?ts$/, "");
```
and remove line 106

Reply:
![image](https://github.com/user-attachments/assets/7048394f-3a19-4bef-9e08-9552c146992d)
I ended up applying only part of it, because the `cts` needed to be preserved. :)